test/operatorStage2.js

```diff
@@ -14,11 +13,6 @@ beforeEach(async () => {
   SEuro = await SEuroContract.deploy('sEURO', 'sEUR', [owner.address]);
   USDT = await ERC20Contract.deploy('USDT', 'USDT', ethers.utils.parseEther('10000000'));
   TST = await ERC20Contract.deploy('TST', 'TST', ethers.utils.parseEther('10000000'));
-  USDT_ADDRESS = USDT.address;
-  TST_ADDRESS = TST.address;
-  SEUR_ADDRESS = SEuro.address;
-  CUSTOMER_ADDR = customer.address;
-  OWNER_ADDR = owner.address;
```

Comment:
sorry i had to remove these constants. there are some issues with the order in which each `beforeEach` is caused, which means that the contract dependencies are inconsistent in the test